src/plasma/Makefile

```diff
@@ -35,8 +35,8 @@ common: FORCE
 	git submodule update --init --recursive
 	cd ../common; make
 
-# Set the request timeout low for testing purposes.
-test: CFLAGS += -DRAY_TIMEOUT=50
+# Set the request timeout low and logging level at FATAL for testing purposes.
+test: CFLAGS += -DRAY_TIMEOUT=50 -DRAY_COMMON_LOG_LEVEL=4
```

Comment:
I wanted to keep the level at FATAL because some of the tests test error scenarios, which is why we have so many log messages right now.